mflowgen/test/test_module.sh

```diff
@@ -325,15 +325,6 @@ function build_module {
     mkdir $dirname; cd $dirname
     mflowgen run --design $garnet/mflowgen/$modname
     set +x
-
-    # This is currently considered best practice for us I think?
-    if [ "$modname" == "full_chip" ]; then
-        set -x
-        echo "Fetch pre-built RTL from stash"
-        mflowgen stash link --path /home/ajcars/tile-array-rtl-stash/2020-0724-mflowgen-stash-75007d
-        mflowgen stash pull --hash 2fbc7a
-        set +x
-    fi
```

Comment:
So...we don't need this anymore? And / or are we going back to building the RTL from scratch?

Reply:
Yes that is the main point of this PR

Reply:
> Yes that is the main point of this PR

haha yes of course. duh. sorry!